docs/CNS.md

```diff
@@ -3,100 +3,123 @@
 The computational engine for most of HADDOCK3's modules is the [Crystallography and NMR System (CNS)](http://cns-online.org/v1.3/).
 Once you have [cloned the HADDOCK3 repository](INSTALL.md), please follow these instructions to compile a CNS executable that can be used to run HADDOCK.
 
-## 1 Download CNS
+Before starting, make sure you download the system requirements, they are `gcc`, `gfortran` and `csh`. Luckily they are easily available from the package manager, so if you are using Ubuntu (for example):
 
-Downloading CNS requires a license, which academic users can [request for free](http://cns-online.org) (click on the `Download` menu and fill the form). You will be sent a password for the download via e-mail.
-Using the link provided in the same e-mail, download CNS ("Source installer (compilation required on your system)") to the directory in which you want to install it (for example `software`). Navigate to this directory, then uncompress and extract the archive:
+```bash
+sudo apt-get install gcc gfortran csh
+```
```

Comment:
My trouble-shoot was that `gfortran` was not in my PATH.
I had to edit my `.bash_profile`, and add `export PATH="/path/homebrew/bin:$PATH"`.
And make sure the command `which gfortran` was successful

Reply:
yeah, I suggest adding smth like “make sure gfortran is installed properly via `which gfortran`"